functable.c

```diff
@@ -53,6 +53,9 @@ extern uint32_t adler32_ssse3(uint32_t adler, const unsigned char *buf, size_t l
 #ifdef X86_AVX2_ADLER32
 extern uint32_t adler32_avx2(uint32_t adler, const unsigned char *buf, size_t len);
 #endif
+#ifdef POWER8
```

Comment:
`#ifdef POWER8_VSX_ADLER32`

Reply:
The requirements for this optimization are the same as for the slide_hash one, in which I used only POWER8. I don't have a strong opinion about either naming scheme. I'd just like to keep them consistent. I see that ARM uses macros like `ARM_NEON_<FUNCTION>` for most optimized functions. x86 also does that for some, but for `slide_hash`, for example, it uses only `X86_SSE2`. Which one do you think we should maintain for POWER? If the `<PROC>_<FEATURE>_<FUNCTION>` is preferred I can switch here and maybe submit a quick PR to update slide_hash.

Reply:
Yeah I know some of them need to be cleaned up. I think the `<PROC>_<FEATURE>_<FUNCTION>` method is the best to be consistent.

Reply:
I agree. Just switched to `POWER8_VSX_ADLER32`